string person1_id # Name of the person1
string person2_id # Name of the person2
float64 person1_X # Person1 x position
float64 person1_Y # Person1 y position
float64 person2_X # Person2 x position
float64 person2_Y # Person2 y position
